src/UserAgreement/QuizRight.less

```diff
@@ -0,0 +1,14 @@
+.quiz__right {
+  display: inline-block;
+  text-align: right;
+  position: relative;
+  padding-right: 30px;
+  width: 285px;
```

Comment:
would be good to have a percentage here as well

Reply:
I think the right panel's width should be fixed while the left side could be a percentage. But, we should discuss with Vivi on this